README.md

```diff
@@ -86,7 +87,8 @@ PockyBot can be messaged directly with certain commands.
 * `welcome` &mdash; display a welcome message.
 * `ping` &mdash; verify that the bot is alive.
 * `help` &mdash; display a list of available commands.
-* `keywords` &mdash; display the available keywords
+* `keywords` &mdash; display the available keywords.
+* `rotation` &mdash; display the rotation.
```

Comment:
What is a 'rotation'? It's not very self-descriptive for someone who comes across this without context.